system-int-test/test/cmr/system_int_test/ingest/subscription_processing_test.clj

```diff
@@ -49,71 +40,305 @@
     (index/wait-until-indexed)
     result))
 
+(defn- mock-send-subscription-emails
+  [context subscriber-filtered-gran-refs-list]
+  (let
+   [send-update-subscription-notification-time! #'jobs/send-update-subscription-notification-time!]
+    (doseq [subscriber-filtered-gran-refs-tuple subscriber-filtered-gran-refs-list
+            :let [[sub-id subscriber-filtered-gran-refs subscriber-id subscription] subscriber-filtered-gran-refs-tuple]]
+      (when (seq subscriber-filtered-gran-refs)
+        (send-update-subscription-notification-time! context sub-id)))
+    subscriber-filtered-gran-refs-list))
+
+(deftest ^:oracle subscription-job-manual-time-constraint-test
```

Comment:
I have two issues with this test:
1) It is not an integration test and doesn't belong in system-int-tests. With the mock and `with-redefs`, it is a code smell for integration test.
2) It does not really test much other than the metadata-db find-concepts. And `find-concepts` has been tested separately already. 
I think we should either rewrite it as a unit test or just delete it.